husky_components/data-navigation/list-view.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * This file is part of Husky frontend development framework.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ *
+ * @module husky/components/data-navigation
+ */
+
+define([], function() {
+
+    var templates = {
+        list: function() {
+            return [
+                '<ul class="data-navigation-items">',
+                '<% if (!!data.children && !!data.children.length) { %>',
+                '<% _.each(data.children, function(child) { %>',
+                '<li data-id="<%= child.id %>" class="data-navigation-item">',
+                '<% if (!!child.preview) { %>',
+                '<div class="data-navigation-item-thumb" style="background-image: url(\'<%=child.preview.url%>\')"></div>',
+                '<% } else { %>',
+                '<div class="fa-coffee data-navigation-item-thumb" style="margin-top: 10px;padding-left: 10px;"></div>',
+                '<% } %>',
+                '<div class="data-navigation-item-name">',
+                '<%= child[options.nameKey] %>',
+                '<% if (!!child.hasSub) { %>',
+                '<span class="fa-chevron-right data-navigation-item-next"></span>',
+                '<% } %>',
+                '</div>',
+                '</li>',
+                '<% }) %>',
+                '<% } else if (!!data.children && data.children.length === 0) { %>',
+                '<li class="not-selectable">',
+                '<div class="data-navigation-info data-navigation-info-empty">',
+                '<%=options.translates.noData%>',
+                '</div>',
+                '</li>',
+                '<% } else { %>',
+                '<li class="not-selectable">',
+                '<div class="data-navigation-loader-container">',
+                '<div class="spinner">',
+                '<div class="double-bounce1"></div>',
+                '<div class="double-bounce2"></div>',
+                '</div>',
+                '</div>',
+                '</li>',
+                '<% } %>',
+                '</ul>'
```

Comment:
Wrong indentation.

And yeah, @3rfan I know you would like to have that in separate files :-P